go/types/simplify_test.go

```diff
@@ -64,20 +64,6 @@ func TestSimplifyType(t *testing.T) {
 			act := simplifyType(in, intersectStructs)
 			assert.True(exp.Equals(act), "Expected: %s\nActual: %s", exp.Describe(), act.Describe())
 		}
-		testSame := func(t *Type) {
-			test(t, resolveCycleTypes(t))
-		}
-
-		testSame(BlobType)
```

Comment:
I'm not sure I understand. They were tests that we didn't do random bad stuff to types?